dataprofiler/tests/profilers/test_base_option.py

```diff
@@ -17,12 +17,21 @@ class TestBaseOption(unittest.TestCase):
 	def setUpClass(cls):
 		cls.data = Data(data=pd.DataFrame([1, 2]), data_type='csv')
 	
+
+	@classmethod
+	def getOptions(self, **params):
+		return BaseOption(**params)
```

Comment:
Why are the tabs 8 spaces here, but only 4 spaces earlier?

Reply:
I believe GitHub's default tab spacing is 4. The view will change if you use the following link: [https://github.com/capitalone/DataProfiler/pull/51/files?ts=4](https://github.com/capitalone/DataProfiler/pull/51/files?ts=4)

Reply:
I believe int his case we should use 4 spaces, or set the IDE to do 4 spaces when you hit tab.

Reply:
My own IDE does 4 spaces but GitHub does not - I can replace all "\t" with four spaces if that's preferable.

Reply:
Yes, please use spaces. for the IDE, I believe you can set it to convert your tab to spaces. such that it isn't saved as a \t